BookingsApi/BookingsApi.Infrastructure.Services/HearingDtoMapper.cs

```diff
@@ -6,21 +6,22 @@ namespace BookingsApi.Infrastructure.Services
 {
     public static class HearingDtoMapper
     {
-        public static HearingDto MapToDto(Hearing hearing)
+        public static HearingAllocationDto MapToDto(Hearing hearing)
```

Comment:
if we are introducing a new dto, then it's probably best to leave this method as it was (returning a `HearingDto`) and add a new method to map the new dto, like this:

```
public static HearingAllocationDto MapHearingAllocationToDto(Hearing hearing)
        {
            var hearingDto = MapToDto(hearing);
        
            var hearingAllocationDto = (HearingAllocationDto)hearingDto;

            hearingAllocationDto.JudgeDisplayName = hearing.Participants?.FirstOrDefault(p => p.HearingRole.UserRole.IsJudge)?.DisplayName;

            return hearingAllocationDto;
        }
```

that way the existing calls to this method won't receive an object with an empty JudgeDisplayName property

Reply:
done